Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:edge
```

Comment:
This is missing some items from the [spec](https://github.com/codeclimate/spec/blob/master/SPEC.md#packaging)